src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/models/ReportVersionWithDescLatestElapsed.kt

```diff
@@ -0,0 +1,17 @@
+package org.vaccineimpact.orderlyweb.models
+
+import java.beans.ConstructorProperties
+import java.time.Instant
+
+data class ReportVersionWithDescLatestElapsed
+@ConstructorProperties("name", "displayname", "id", "published", "date", "latestVersion", "description", "elapsed")
+constructor(
+    override val name: String,
+    override val displayName: String?,
+    override val id: String,
+    override val published: Boolean,
+    override val date: Instant,
+    val latestVersion: String,
```

Comment:
This is weird, but doesn't seem to be a change from master. I tried a couple of hacks to fix it but couldn't!

Reply:
I think for some reason you have to assert individual fields. I didn't see assertion for latestVersion but the rest of the data fields.

Reply:
This is true - I got as far as this, but not very happy with it as a solution...
```
assertThat(sut.getByNameAndVersion().basicReportVersion.latestVersion).isEqualTo(report.basicReportVersion.latestVersion)
```

Reply:
Thanks both I'll give it a try. As you say, @mwoodbri it's odd that it doesn't seem to complain about other fields.